nfs.ks.in

```diff
@@ -0,0 +1,23 @@
+#test name: nfs
+nfs --server=@KSTEST_NFS_SERVER@ --dir=@KSTEST_NFS_PATH@
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart --all
+autopart
+
+keyboard us
+lang en
+timezone America/New_York
+rootpw qweqwe
+shutdown
+
+%packages
+%end
```

Comment:
I wonder if some of that has a fragment, but it's really and unimportant detail, compared to the scope of this change.

Reply:
This is a leftover of the `nfs-repo-and-addon.ks.in` file, so I don't want to touch it more than is necessary.